NAMESPACE

```diff
@@ -239,6 +239,7 @@ importFrom(ggplot2,ggplot_build)
 importFrom(ggplot2,ggplot_gtable)
 importFrom(ggplot2,labs)
 importFrom(ggplot2,position_dodge)
+importFrom(ggplot2,position_stack)
```

Comment:
I was just about to comment on this :)

There should also be a roxygen2 tag in `pixelatorR-package.R`. The easiest way to add imported functions is to do:

````
usethis::use_import_from("ggplot2", "position_stack")
````

Then run `devtools::document()` to add the import to `NAMESPACE`.